app/jobs/fetch_faculty_data_job.rb

```diff
@@ -18,6 +18,7 @@ class FetchFacultyDataJob < ActiveJob::Base
   }
 
   def perform
+    Faculty.delete_all
```

Comment:
I'd recommend a comment here and/or in model noting that Faculty records are never modified within the application. That could help future devs -- especially if someone unwittingly adds a feature that updates faculty records in the application without looking at this file and has to figure out why updates are being wiped out every night.